config/tests/samples/create/contents_test.go

```diff
@@ -15,11 +12,12 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-var matchEverythingRegex = regexp.MustCompile(".*")
-
 func TestNames(t *testing.T) {
-	project := testgcp.GetDefaultProject(t)
-	samples := loadSamplesOntoUnstructs(t, matchEverythingRegex, project)
+	project := testgcp.GCPProject{
+		ProjectID:     "tests-testnames",
```

Comment:
Not a big deal but it might be nice if the fake project name contained "project"?
Makes it easier to find/understand where it came from if it shows up somewhere unexpected.